src/Middleware/CORS/src/Internal/CORSLoggerExtensions.cs

```diff
@@ -25,47 +25,47 @@ static CORSLoggerExtensions()
         {
             _isPreflightRequest = LoggerMessage.Define(
                 LogLevel.Debug,
-                1,
+                 new EventId(1, "IsPreflightRequest"),
```

Comment:
BTW the formatting seems a bit off - the new event id lines are indented one extra space?

Reply:
It's done.